helm/app/templates/pg-cluster-pdb.yaml

```
@@ -0,0 +1,10 @@
+# We may have to remove this and use crunchy minAvailable once the operator is upgraded
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: hippo-ha-pdb
```

all of our objects should have these labels so that we can easily identify the objects managed by helm: 
```suggestion
  name: hippo-ha-pdb
  labels: {{ include "ccbc.labels" . | nindent 4 }}
```